src/test/FuzzerImpl.cpp

```diff
@@ -838,70 +898,569 @@ applyFuzzOperations(LedgerTxn& ltx, PublicKey const& sourceAccount,
+    }
+
+    constexpr TrustLineParameters(AssetID assetID, uint32_t allowTrustFlags,
+                                  int64_t assetAvailableForTestActivity,
+                                  int64_t spareLimitAfterSetup, int sponsorKey,
+                                  ClaimableBalances const& claimableBalances)
+        : SponsoredEntryParameters(sponsorKey)
+        , mAssetID(assetID)
+        , mAllowTrustFlags(allowTrustFlags)
+        , mAssetAvailableForTestActivity(assetAvailableForTestActivity)
+        , mSpareLimitAfterSetup(spareLimitAfterSetup)
+        , mClaimableBalances(claimableBalances)
+    {
+        assert(assetID.mIssuer < FuzzUtils::NUMBER_OF_PREGENERATED_ACCOUNTS);
+    }
+
+    // Construct an inactive (not-to-be-created) trust line.
+    constexpr TrustLineParameters()
+        : SponsoredEntryParameters()
+        , mAssetID()
+        , mAllowTrustFlags(0)
+        , mAssetAvailableForTestActivity(0)
+        , mSpareLimitAfterSetup(0)
+        , mClaimableBalances(
+              {ClaimableBalanceParameters(), ClaimableBalanceParameters()})
+    {
+    }
+
+    bool
+    isActive() const
+    {
+        return !mAssetID.mIsNative;
+    }
+
+    AssetID const mAssetID;
+    uint32_t const mAllowTrustFlags;
+    int64_t const mAssetAvailableForTestActivity;
+    int64_t const mSpareLimitAfterSetup;
+    ClaimableBalances const mClaimableBalances;
+};
+
+auto constexpr MAX_TRUST_LINES_PER_ACCOUNT = 4;
+
+struct AccountParameters : public SponsoredEntryParameters
+{
+    // A trust line with a native asset parameter represents an empty entry --
+    // but we use std::array with empty entries rather than std::vector because
+    // the latter can not be constexpr until C++20.
+    using TrustLines =
+        std::array<TrustLineParameters, MAX_TRUST_LINES_PER_ACCOUNT>;
+
+    constexpr AccountParameters(int64_t nativeAssetAvailableForTestActivity,
+                                uint32_t optionFlags,
+                                TrustLines const& trustLines)
+        : SponsoredEntryParameters()
+        , mNativeAssetAvailableForTestActivity(
+              nativeAssetAvailableForTestActivity)
+        , mOptionFlags(optionFlags)
+        , mTrustLines(trustLines)
+    {
+    }
+
+    constexpr AccountParameters(int64_t nativeAssetAvailableForTestActivity,
+                                int sponsorKey, uint32_t optionFlags,
+                                TrustLines const& trustLines)
+        : SponsoredEntryParameters(sponsorKey)
+        , mNativeAssetAvailableForTestActivity(
+              nativeAssetAvailableForTestActivity)
+        , mOptionFlags(optionFlags)
+        , mTrustLines(trustLines)
+    {
+    }
+
+    int64_t const mNativeAssetAvailableForTestActivity;
+    uint32_t const mOptionFlags;
+    TrustLines const mTrustLines;
 };
 
-struct OfferParameters
+auto constexpr DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY = 256;
```

Comment:
The name of this variable makes me think it's an asset instead of an amount. I think this should be renamed to `DEFAULT_AMOUNT_AVAILABLE_FOR_TEST_ACTIVITY` or something similar.

Reply:
I agreed.  Then I went further and decided that the constant was making things less readable, not more, and the consistency that it was creating wasn't necessary, so I simply replaced the constant with its literal value altogether (with the intent of varying the literal values next).